modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/search/CommerceOrderSearchRegistrar.java

```diff
@@ -41,8 +41,6 @@ protected void activate(BundleContext bundleContext) {
 					_modelIndexWriterContributor);
 				modelSearchDefinition.setModelSummaryContributor(
 					_modelSummaryContributor);
-				modelSearchDefinition.setSearchResultPermissionFilterSuppressed(
-					true);
 			});
```

Comment:
why do you need to remove this?

Reply:
Because this way they no longer suppress the permission filter